src/messaging/ReliableMessageMgr.cpp

```diff
@@ -189,9 +189,9 @@ void ReliableMessageMgr::ExecuteActions()
             ExchangeHandle ec(entry->ec);
 
             ChipLogError(ExchangeManager,
-                         "<<%d [E:" ChipLogFormatExchange " S:%u M:" ChipLogFormatMessageCounter
-                         "] (%s) Msg Retransmission to %u:" ChipLogFormatX64 " failure (max retries:%d)",
-                         sendCount + 1, ChipLogValueExchange(&entry->ec.Get()), session->SessionIdForLogging(), messageCounter,
+                         "<<_ [E:" ChipLogFormatExchange " S:%u M:" ChipLogFormatMessageCounter
```

Comment:
Why do we need the "<<_" bit at all?

Reply:
We probably don't I just wanted to remove the count of there. There some script that parse the logs in the repo and I saw it use << to catch those traces so I kept that in case it needed this particular trace still